packages/create-redwood-app/template/scripts/seed.ts

```diff
@@ -20,7 +19,7 @@ export default async () => {
       // { name: 'bob', email: 'bob@example.com' },
     ]
     console.log(
-      "\nUsing the default './scripts/seed.js' template\nEdit the file to add seed data\n"
+      "\nUsing the default './scripts/seed.ts' template\nEdit the file to add seed data\n"
```

Comment:
js|ts will be respective to the project language target. Should we do something like this here?
```
'./scripts/seed.{js,ts}'
```

Isn't that the "official" syntax for the extension option?

Reply:
Yeah that would be more accurate. And yeah that's the correct syntax for path expansion. I can PR this real quick.

Reply:
Oh you did it already; nice